pkg/config/piped.go

```diff
@@ -345,6 +351,13 @@ func (g PipedGit) LoadSSHKey() ([]byte, error) {
 	return nil, errors.New("either sshKeyFile or sshKeyData must be set")
 }
 
+func (g *PipedGit) Validate() error {
+	if g.ShouldConfigureSSHConfig() && g.PersonalAccessToken.Validate() {
+		return errors.New("cannot configure both sshKeyData or sshKeyFile and personalAccessToken")
+	}
+	return nil
+}
```

Comment:
Can you add a unit test for this function?
I don't feel this is not enough. What about a case in which one of `ShouldConfigureSSHConfig` and `PersonalAccessToken.Validate` returns false?

Reply:
Corrected missing test case!